src/shared/components/palette/organisation/write-organisation-field.component.ts

```diff
@@ -29,8 +35,11 @@ export class WriteOrganisationFieldComponent extends AbstractFieldWriteComponent
   public simpleOrganisations$: Observable<SimpleOrganisationModel[]>;
   public selectedOrg$: Observable<SimpleOrganisationModel>;
 
-  constructor(private organisationService: OrganisationService, private organisationConverter: OrganisationConverter) {
+  constructor(private organisationService: OrganisationService,
```

Comment:
All of these parameters should be `readonly`.

Reply:
done